user-guide/toc.yml

```diff
@@ -1945,6 +1934,80 @@
+            - name: Query data sources
+              topicUid: Query_data_sources
+            - name: Query operators
+              topicUid: Query_operators
+              items:
+                - name: Aggregate
+                  topicUid: GQI_Aggregate
+                - name: Column manipulations
+                  topicUid: GQI_Column_manipulations
+                - name: Filter
+                  topicUid: GQI_Filter
+                - name: Join
+                  topicUid: GQI_Join
+                - name: Select
+                  topicUid: GQI_Select
+                - name: Sort
+                  topicUid: GQI_Sort
+                - name: Sort by
+                  topicUid: GQI_Sort_by
+                - name: Then sort by
+                  topicUid: GQI_Then_sort_by
+                - name: Top X
+                  topicUid: GQI_Top_X
+            - name: Extensions
+              topicUid: Configuring_an_ad_hoc_data_source_in_a_query
+              items:
+                - name: Configuring an ad hoc data source
+                  topicUid: Configuring_an_ad_hoc_data_source_in_a_query
+                  items:
+                    - name: Building blocks of queries with ad hoc data
+                      topicUid: Ad_hoc_Building_blocks
+                    - name: Life cycle of queries with ad hoc data
+                      topicUid: Ad_hoc_Life_cycle
+                    - name: Tutorials
+                      topicUid: Ad_hoc_Tutorials
+                - name: Configuring a custom operator
+                  topicUid: GQI_Custom_Operator
+                  items:
+                    - name: Building blocks of a custom operator
+                      topicUid: CO_Building_blocks
+                    - name: Life cycle of a custom operator
+                      topicUid: CO_Life_cycle
+                - name: API reference
+                  topicUid: IGQIDataSource
+                  items:
+                    - name: IGQIDataSource
+                      topicUid: IGQIDataSource
+                    - name: IGQIInputArguments
+                      topicUid: IGQIInputArguments
+                    - name: IGQIOnInit
+                      topicUid: IGQIOnInit
+                    - name: IGQIOnPrepareFetch
+                      topicUid: IGQIOnPrepareFetch
+                    - name: IGQIOnDestroy
+                      topicUid: IGQIOnDestroy
+                    - name: GQIColumn
+                      topicUid: GQIColumn
+                    - name: GQIPage
+                      topicUid: GQIPage
+                    - name: GQIRow
+                      topicUid: GQIRow
+                    - name: GQICell
+                      topicUid: GQICell
+                    - name: GQIArgument
+                      topicUid: GQIArgument
+                    - name: GQIDMS
+                      topicUid: GQIDMS
+            - name: Importing a query
+              topicUid: Importing_a_query
```

Comment:
Looking around, this most closely fits here: [Applying a data feed](https://docs.dataminer.services/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Dashboards_app/Creating_and_configuring_dashboards/Configuring_dashboard_components.html#applying-a-data-feed).
This section could maybe also be its own page where the different data feeds can be listed individually with their own specifics.

Reply:
Thanks for the suggestion! So, is importing a query limited to the Dashboards app then? If this information also pertains to Low-Code Apps, it would be illogical to move this page under the "Dashboards app" section.

Reply:
I agree that would not be very logical.

For now, it does look like importing queries in Low-Code Apps is not supported, but I would imagine that this is a feature we will also want to support there in the future.
That said, "Applying a data feed" is currently part of the "Dashboard app" section, but that does also apply to the Low-Code Apps.
So it's already not ideal.

Reply:
We propose renaming "Importing a query" to "Reusing a query in a different dashboard" to clarify that the query is being imported into a dashboard, not GQI itself. From a user-perspective, we believe it's best to keep this page within the GQI documentation since it is the only place where queries are mentioned. Moving this information to "Applying a data feed" may confuse users and make it difficult for them to find or realize that this option exists. And as you said, the information in "Applying a data feed" also applies to the Low-Code Apps, while the information on importing a query (for now) does not.

Reply:
I would suggest to create a section in [Applying a data feed](https://docs.dataminer.services/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Dashboards_app/Creating_and_configuring_dashboards/Configuring_dashboard_components.html#applying-a-data-feed) that solely relates to creating, editing and importing queries.

So perhaps we can merge this as it is and then in a future change move this section?

Reply:
I would also order the items in the API reference alphabetically.

Reply:
> I would suggest to create a section in [Applying a data feed](https://docs.dataminer.services/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Dashboards_app/Creating_and_configuring_dashboards/Configuring_dashboard_components.html#applying-a-data-feed) that solely relates to creating, editing and importing queries.
> 
> So perhaps we can merge this as it is and then in a future change move this section?

OK, I will also add this to the to-do list of the second revision then!

Reply:
> I would also order the items in the API reference alphabetically.

Implemented in [commit 380ff9e](https://github.com/SkylineCommunications/dataminer-docs/pull/1694/commits/380ff9ef3a36910c4716e5e659e09f1d398b76fd).